src/tools/sssctl/sssctl_logs.c

```diff
@@ -545,15 +544,15 @@ errno_t sssctl_debug_level(struct sss_cmdline *cmdline,
     struct debuglevel_tool_ctx *ctx = NULL;
     struct poptOption long_options[] = {
         {"domain", '\0', POPT_ARG_ARGV, &pc_domains,
-            0, _("Target a specific domain"), NULL},
-        POPT_SERV_OPTION(SSSD, pc_services),
-        POPT_SERV_OPTION(NSS, pc_services),
-        POPT_SERV_OPTION(PAM, pc_services),
-        POPT_SERV_OPTION(SUDO, pc_services),
-        POPT_SERV_OPTION(AUTOFS, pc_services),
-        POPT_SERV_OPTION(SSH, pc_services),
-        POPT_SERV_OPTION(PAC, pc_services),
-        POPT_SERV_OPTION(IFP, pc_services),
+            0, _("Target a specific domain"), _("domain")},
+        POPT_SERV_OPTION(SSSD, pc_services, "Target the SSSD service"),
+        POPT_SERV_OPTION(NSS, pc_services, "Target the NSS service"),
+        POPT_SERV_OPTION(PAM, pc_services, "Target the PAM service"),
+        POPT_SERV_OPTION(SUDO, pc_services, "Target the SUDO service"),
+        POPT_SERV_OPTION(AUTOFS, pc_services, "Target the AUTOFS service"),
+        POPT_SERV_OPTION(SSH, pc_services, "Target the SSH service"),
+        POPT_SERV_OPTION(PAC, pc_services, "Target the PAC service"),
+        POPT_SERV_OPTION(IFP, pc_services, "Target the IFP service"),
```

Comment:
All those strings should be localized (`_()`), I think?

Reply:
I mean, if we translate "Target a specific domain" then we should translate other similar strings.

Reply:
They are. The `_()` macro is called from inside the macro `POPT_SERV_OPTION()`.

Reply:
Either I did something wrong or
```
$ grep "Target " sssd.pot 
msgid "Target a specific domain"
```

Reply:
I did test it, but now it doesn't work. That probably means I tested it and later I made a last-minute-change that ruined it.

On the other side, my guess is that `po4a` runs before the preprocessor, otherwise the initial problem that trigger this PR would have never happened. That also explain why you just saw it not working.

I moved the `_()` macro out of the `POPT_SERV_MACRO()` and checked it works right before pushing the changes.